src/api/search/index.ts

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from 'react-query';
+import useGeolocation from 'utils/hooks/useGeolocation';
+import searchApi from './searchApiClient';
+
+export function useTrendingQuery() {
+  const {
+    isLoading, isError, data,
+  } = useQuery('trending', () => searchApi.get('/trending'));
+  const trendings = data?.data.trendings;
+  return {
+    isLoading, isError, trendings,
+  };
+}
+
+export function useShopQuery(place_id: string) {
+  const {
+    isLoading, isError, data,
+  } = useQuery('shop', () => searchApi.get(`/shop?place_id=${place_id}`));
+  return {
+    isLoading, isError, data,
+  };
+}
```

Comment:
기존의 useQuery가 반환하는 값에서 더 한정시켜서 얻을 수 있는 장점이 없어보여요.
그리고 user쪽이랑 비교하게 되는데, 그쪽은 함수 표현식으로 작성이 되어있네요? `utils/hooks`는 또 선언식이고..
컨벤션을 정할 필요가 있어보이네요 ㅋㅋ 
아, 그리고 Query Key가 누락됐다고 생각해요.
```suggestion
export const useShopQuery = (place_id: string) => useQuery(['shop', place_id], () => searchApi.get(`/shop?place_id=${place_id}`));
```

Reply:
맞네요 `/shop` 요청은 하나의 상점에 대한 요청이라 상점의 `place_id`가 변하면 새로운 요청을 해줘야 하니... 
쿼리 키에 대해 다시 공부하는 좋은 시간이었습니다 ㅋㅋㅋ 
쿼리 키 추가하겠습니다~~~